src/uwtools/j2template.py

```diff
@@ -69,98 +25,105 @@ def __init__(
         **kwargs,
     ) -> None:
         """
-        Parameters
-        ----------
-        configure_obj
-            See class-level docstring
-        template_path
-            Path to a Jinja2 template file
-        template_str
-            A Jinja2 template
-
-        Keyword Arguments
-        -----------------
-        loader_args : dict
-            A dictionary of arguments to pass to the J2 loader
-        log_name : str
-            The name of the logging object to be used.
+        :param configure_obj: Key/value pairs needed to render the provided template.
+        :param template_path: Path to a Jinja2 template file.
+        :param template_str: An in-memory Jinja2 template.
+        :raises RuntimeError: If neither a template file or path is provided.
```

Comment:
As elsewhere below, Sphinx-style param/raises/return entries have been added to methods/functions, or existing entries updated for this style. We'll be working in the future on extending this and automatically generating HTML docs from these annotations.